pkg/externalauthprovider/flags.go

```diff
@@ -79,14 +79,14 @@ func (e *ExternalAuthServiceImpl) CreateExternalAuthProvider(cluster *cmv1.Clust
 	externalAuthConfig, err := CreateExternalAuthConfig(args)
 	if err != nil {
 		return fmt.Errorf("failed to create an external authentication provider for cluster '%s': %s",
-			cluster.ID(), err)
+			cluster.Name(), err)
```

Comment:
I think we should show the clusterKey provided by the customer instead, this way it will show what the customer uses to identify the cluster. What do you think?

Reply:
Thanks @gdbranco agreed, let me update it